detecthttp/pytest.py

```diff
@@ -70,11 +70,16 @@ def pytest_runtest_call(item):
 
     if enabled:
         cassette_manager.__exit__(*sys.exc_info())
-        if len(cassette):
+        requested_host = []
+        for request in cassette.data:
+            domain = request[0].host.replace('www.', '')  # remove www if exists
```

Comment:
should we first check if `request[0].host` is empty  for attempting string manipulation?

Reply:
I didn't think it's necessary because if exists, is a tuple of request/response recorded by vcr:

https://github.com/kevin1024/vcrpy/blob/master/vcr/cassette.py#L238